sdk/identity/azure-identity/pom.xml

```diff
@@ -17,6 +17,7 @@
     <!-- Configures the Java 9+ run to perform the required module exports, opens, and reads that are necessary for testing but shouldn't be part of the module-info. -->
     <javaModulesSurefireArgLine>
       --add-opens java.xml/jdk.xml.internal=ALL-UNNAMED
+      --add-opens com.azure.core/com.azure.core.implementation.util=ALL-UNNAMED<!-- Powermock seems to be messing up modules, i.e. NettyAsyncHttpClient for some reason ends up in unnamed module in runtime  -->
```

Comment:
@g2vinay @alzimmermsft any idea why this is happening only in identity? I suspect it's powermock doing but I don't have solid proof other than inability to find better explanation.

![image](https://user-images.githubusercontent.com/61715331/157118792-1d2aa82a-b136-4434-9ede-f46f05a99b17.png)

Reply:
Let's work on getting PowerMock removed as it won't work when Java 17 is used anyways